src/middlewares/logger.middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable, Logger, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
```

Comment:
Just a FYI, we use fastify because of low overhead. This line will be different for rumsan seed.